packages/material-ui/src/Divider/Divider.js

```diff
@@ -20,14 +22,28 @@ export const styles = theme => ({
     left: 0,
     width: '100%',
   },
-  /* Styles applied to the root element if `inset={true}`. */
+  /* Styles applied to the root element if `variant={'inset'}`. */
   inset: {
     marginLeft: 72,
+    '& + $subheader': {
+      marginLeft: 72,
```

Comment:
Duplicate style?

Reply:
![image](https://user-images.githubusercontent.com/12938082/48446526-62e13280-e791-11e8-837f-e42df175d8d1.png)
![image](https://user-images.githubusercontent.com/12938082/48446548-6d9bc780-e791-11e8-88dd-165da5eef6ec.png)

Middle and inset dividers affect the subheader differently I wasn't sure on how to handle this so I added for the inset divider here so it overrides the default